histomicstk/features/FeatureExtraction.py

```diff
@@ -1,12 +1,15 @@
-import collections
-import math
 import numpy as np
 import pandas as pd
-from scipy.stats import entropy, skew, kurtosis
 from skimage.feature import canny
 from skimage.measure import regionprops
-from skimage.segmentation import find_boundaries
-from skimage.morphology import disk, dilation
+from skimage.morphology import disk, binary_dilation
+
+from .ComputeFSDFeatures import ComputeFSDFeatures
+from .ComputeGradientFeatures import ComputeGradientFeatures
+from .ComputeIntensityFeatures import ComputeIntensityFeatures
+from .ComputeMorphometryFeatures import ComputeMorphometryFeatures
```

Comment:
I think we can go with 'shape' instead of 'morphometry'. Although morphometry strictly means shape, in pathology it also is used to describe textural and intensity characteristics.